market-contract/tests/utils/setup.rs

```diff
@@ -7,14 +7,15 @@ use fuels::{
     types::Identity,
 };
 
+// PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("market-contract/out/debug/market-contract.bin");
+const MARKET_CONTRACT_BINARY_PATH: &str = "./market-contract/out/debug/market-contract.bin";
+const MARKET_CONTRACT_STORAGE_PATH: &str =
+    "./market-contract/out/debug/market-contract-storage_slots.json";
```

Comment:
```suggestion
const MARKET_CONTRACT_BINARY_PATH: &str = "../market-contract/out/debug/market-contract.bin";
const MARKET_CONTRACT_STORAGE_PATH: &str =
    "../market-contract/out/debug/market-contract-storage_slots.json";
```

This will make the tests run